ReactiveCocoa/DelegateProxy.swift

```diff
@@ -53,57 +53,64 @@ extension DelegateProxy {
 	internal static func proxy<P: DelegateProxy<Delegate>>(
 		for instance: NSObject,
 		setter: Selector,
-		getter: Selector,
-		_ key: StaticString = #function
+		getter: Selector
 	) -> P {
-		return _proxy(for: instance, setter: setter, getter: getter, key) as! P
+		return _proxy(for: instance, setter: setter, getter: getter) as! P
 	}
 
 	private static func _proxy(
```

Comment:
I think we could unify the proxy could here with the proxy code in `ActionProxy`.